Cargo.toml

```diff
@@ -19,7 +19,7 @@ serde = { version = "1.0.216", features = ["derive"] }
 strum = { version = "0.26.3", features = ["derive", "phf"] }
 thiserror = "2.0.11"
 time = "0.3.37"
-tokio = { version = "1.42.0", features = ["fs", "io-util"], optional = true }
+tokio = { version = "1.42.0", features = ["fs", "io-util", "rt"], optional = true }
```

Comment:
Why is this addition necessary?  If it's just for testing, move to dev dependencies.